docs/6.bridge/5.nuxt3-compatible-api.md

```diff
@@ -1,112 +1,10 @@
-# Composition API
+# Nuxt 3 compatible API
```

Comment:
What about something like this to make it clearer it's in opposition to 'Legacy Composition API'?

```suggestion
# New Composition API
```

Reply:
I think it's good!